packages/studio-base/src/panels/Plot/useDatasets.ts

```diff
@@ -184,6 +188,18 @@ function useData(id: string, params: PlotParams) {
       clients = rest;
       chooseClient();
     };
+  }, [id]);
+
+  useEffect(() => {
+    const { [id]: client } = clients;
+    if (client == undefined) {
+      return;
+    }
+
+    clients = {
+      ...clients,
+      [id]: { ...client, params },
+    };
```

Comment:
```suggestion
    };
    chooseClient();
```

This seems to fix some of the newly-blank stories locally. But I'm not sure it is a totally correct change.